content/en/docs/ops/integrations/spire/index.md

```diff
@@ -0,0 +1,385 @@
+
+   Entry ID         : f0544fd7-1945-4bd1-88dc-0a5513fdae1c
+   SPIFFE ID        : spiffe://example.org/ns/spire/sa/spire-agent
+   Parent ID        : spiffe://example.org/spire/server
+   Revision         : 0
+   TTL              : default
+   Selector         : k8s_psat:agent_ns:spire
+   Selector         : k8s_psat:agent_sa:spire-agent
+   Selector         : k8s_psat:cluster:demo-cluster
+   {{< /text >}}
+
+* Check Ingressgateway pod state:
+
+{{< text bash >}}
+$ kubectl get pods -n istio-system
+NAME                                    READY   STATUS    RESTARTS   AGE
+istio-ingressgateway-5b45864fd4-lgrxs   1/1     Running   0          60s
+istiod-989f54d9c-sg7sn                  1/1     Running   0          45s
+{{< /text >}}
+
+After registering an entry for Ingressgateway pod, Envoy receives the identity issued by SPIRE and uses it for all TLS and mTLS 
+communications.
+
+### Check Certificate
+
+* Get httpbin service address:
+
+{{< text bash >}}
+$ kubectl get service httpbin
+NAME      TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)    AGE
+httpbin   ClusterIP   10.96.55.188   <none>        8000/TCP   40s
+{{< /text >}}
+
+* Verify that the httpbin certificate was issued by SPIRE.
+
+{{< text bash >}}
+$ kubectl exec $INGRESS_POD -n istio-system -- bash -c "openssl s_client -showcerts -connect 10.96.55.188:8000"
+{{< /text >}}
+
+   {{< text >}}
+   ...
+    ---
+    Certificate chain
+    0 s:C = US, O = SPIRE, CN = httpbin-5f4d47c948-zm6st
+    i:C = US, O = SPIFFE
+    -----BEGIN CERTIFICATE-----
+    MIIC8DCCAdigAwIBAgIRANo+nhRtC6kTnbLUz1MelwcwDQYJKoZIhvcNAQELBQAw
+    HjELMAkGA1UEBhMCVVMxDzANBgNVBAoTBlNQSUZGRTAeFw0yMjAzMTAwMjMyMTVa
+    Fw0yMjAzMTAwMzMyMjVaMEAxCzAJBgNVBAYTAlVTMQ4wDAYDVQQKEwVTUElSRTEh
+    MB8GA1UEAxMYaHR0cGJpbi01ZjRkNDdjOTQ4LXptNnN0MFkwEwYHKoZIzj0CAQYI
+    KoZIzj0DAQcDQgAEMveAwlgnO7YjRZ4vKsAkCYZu3W0REs1nKu/XPEWEBzfht3hl
+    grziL3mLnECWeDnTdakjTnowESCsg3cidOpWEaOB0TCBzjAOBgNVHQ8BAf8EBAMC
+    A6gwHQYDVR0lBBYwFAYIKwYBBQUHAwEGCCsGAQUFBwMCMAwGA1UdEwEB/wQCMAAw
+    HQYDVR0OBBYEFHnxGd7U2ADDw4GouBb0wPLorrHeMB8GA1UdIwQYMBaAFLnNFhHO
+    0Xsdte9uf3VLILKA7TGtME8GA1UdEQRIMEaCGGh0dHBiaW4tNWY0ZDQ3Yzk0OC16
+    bTZzdIYqc3BpZmZlOi8vZXhhbXBsZS5vcmcvbnMvZGVmYXVsdC9zYS9odHRwYmlu
+    MA0GCSqGSIb3DQEBCwUAA4IBAQA6UZagZrw/htbpD7HbcmD17sQ2xv1+nMrRFZGZ
+    M9bO924uyz9MiEf/iR3jKD+wXrJdmyjJ+5Mas4QGnggaGUfDmE2wrVlFKIqY8R1E
+    hZonQm8YeWN4CtDijyAxHihgKTV2V2inrHaaqp1r2/Upt3/4XrZf1f1eqpl03hT9
+    +OPu0SV8MUPTPAXqaIPfjmV9fTTMYQg4g4smsijVKAl6lVs+PsZiqbcgEeUnT2JL
+    lel5nNXdXqKhygfqKesI30BERGPvZ5vdMtUrGxe0L5CPJrkcrtn/vRTuMsXWSr4m
+    VAw3TEEXVdnd3VuAIsKO+6J7biMZH6jhm/sPUqTrdjH/L7S7
+    -----END CERTIFICATE-----
+    ---
+    Server certificate
+    subject=C = US, O = SPIRE, CN = httpbin-5f4d47c948-zm6st
+    
+    issuer=C = US, O = SPIFFE
+   ...
+   {{< /text >}}
+
+## SPIFFE Federation
+
+SPIRE Servers are able to authenticate SPIFFE identities originated from different trust domains, this is known as SPIFFE federation.

```

Comment:
Here we can link to either the guide on spiffe.io on [Deploying SPIFFE Federation Architecture](https://spiffe.io/docs/latest/architecture/federation/readme/) or the [SPIFFE Federation spec](https://github.com/spiffe/spiffe/blob/main/standards/SPIFFE_Federation.md). I guess the latter would make more sense but don't feel strongly about it.

Reply:
I think maybe the first, as it looks more related to what we will cover in this section? (as they're both instructions). Probably not a strong argument as well, just trying to add something 😛